astropy/stats/tests/test_funcs.py

```diff
@@ -9,12 +9,19 @@
 from numpy.testing import assert_equal
 from numpy.testing.utils import assert_allclose
 
+from astropy.utils import minversion
+
 try:
     import scipy  # pylint: disable=W0611
 except ImportError:
     HAS_SCIPY = False
+    HAS_SCIPY_0140 = False
 else:
     HAS_SCIPY = True
+    if minversion(scipy, '0.14.0'):
+        HAS_SCIPY_0140 = True
```

Comment:
By analogy with what is done in `utils.compat.numpycompat`, I would use the term `SCIPY_LT_1_4`.  Trying to be consistent will help with eventually removing those types of statements.